code/iaas/bootstrap/src/main/java/io/cattle/platform/bootstrap/script/BootstrapScriptsHandler.java

```diff
@@ -1,27 +1,54 @@
 package io.cattle.platform.bootstrap.script;
 
+import io.cattle.platform.archaius.util.ArchaiusUtil;
 import io.cattle.platform.iaas.api.request.handler.ScriptsHandler;
+import io.cattle.platform.token.impl.RSAKeyProvider;
 import io.github.ibuildthecloud.gdapi.request.ApiRequest;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
+import java.io.InputStream;
+import java.security.cert.Certificate;
+
+import javax.inject.Inject;
 
 import org.apache.commons.io.IOUtils;
 
+import com.netflix.config.DynamicStringProperty;
+
 public class BootstrapScriptsHandler implements ScriptsHandler {
 
+    private static final DynamicStringProperty BOOTSTRAP_SOURCE = ArchaiusUtil.getString("bootstrap.source");
+    private static final DynamicStringProperty REQUIRED_IMAGE = ArchaiusUtil.getString("bootstrap.required.image");
+
     private static final String BOOTSTRAP = "bootstrap";
 
+    @Inject
+    RSAKeyProvider keyProvider;
+
     @Override
     public boolean handle(ApiRequest request) throws IOException {
         if (!BOOTSTRAP.equals(request.getId())) {
             return false;
         }
 
-        byte[] content = BootstrapScript.getBootStrapSource(request);
```

Comment:
This makes the mehtods on BootstrapScript class no longer used. Why not delete them (and potentially the whole class if you can use a different one for the class loader below)